docs/examples/off_axis_propagation.ipynb

```diff
@@ -287,6 +242,76 @@
     "axs[1].set_xlabel(\"x (microns)\");"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default `use_czt` is set to `False`, as using it is slower than the method mentioned in the above paper. Below we profile the two methods."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Avg time for BLAS with adjusted kernel:  0.1020186033193022\n",
+      "Avg time for BLAS with CZT:  0.21877838999964297\n"
+     ]
+    }
+   ],
+   "source": [
+    "## profiling and checking jit compilation\n",
```

Comment:
@diptodip @GJBoth, since notebooks are annoying with version control, here's the main change 😉